news/_posts/2023-01-10-playtest-20230110.md

```diff
@@ -0,0 +1,119 @@
+---
+kind: article
+title: "Playtest 20230110"
+author: "PunkPun"
+created_at: 2023-01-10 23:00 +0000
+disqus_id: "playtest-20221223"
+permalink: "/news/playtest-20230110/"
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+We are kicking off New Year with a new playtest!
+
+Notable changes in [Playtest 20230110]({{ '/download' | relative_url }}) include:
+
+* Fixed sound device detection on linux
+* Fixed Discord game invites
+* Fixed replays locking up when one of the players dropped connection
+* Fixed Dune 2000 service depot being uncapturable
+* Removed crashing tiles from Red Alert temperate tileset bridge template
+* Other minor fixes
```

Comment:
We already write "changes include" above, implying that this list isn't complete. So this line can/should be removed. ("Other minor fixes" isn't really a "notable change".)

Reply:
I agree